hooks/shared/useCustomToast/type.ts

```diff
@@ -0,0 +1,11 @@
+export const TOAST_TYPE = {
+  SUCCESS: 'success',
+  ERROR: 'error',
+} as const;
+
+export type ToastType = (typeof TOAST_TYPE)[keyof typeof TOAST_TYPE];
+
+export interface CustomToastProps {
+  message: string;
+  type: ToastType;
+}
```

Comment:
타입 파일에서 상수가 같이 있는건 약간 어색한 것 같아서
단순하게 특정 string으로만 한정하는 건 어떠신가요~?
어짜피 ts가 에러는 잡아주니깐요!

그리고 interface, type 둘 중 하나만 쓰는 건 어떠실지요?!

```suggestion
export type ToastType = 'success' | 'error';

export type CustomToastProps = {
  message: string;
  type: ToastType;
};
```

Reply:
아하 넵넵 그렇게할게요~~!!! 사실 상수를 따로 빼둘까 고민했는데 그러기엔 너무 보기 불편해서 8ㅅ8;; 그냥 string으로 두는게 낫겠네요!